### DISCUSSION TITLE
LOTW export

### DISCUSSION THREAD

**Discussion by Halfsat** [magicbug/Cloudlog#992] [Q&A] [answered]
First, Thanks for all the help provided here. I have managed to get just about everything up and running on my CloudLog server. Except LOTW. Going to the Admin> LOTW import/export, I only have the option to import. And I haven't found the window shown in the documentation to upload the cert or see the status. I must be missing something simple?
![Screen Shot 2021-04-13 at 12 36 49 AM](https://user-images.githubusercontent.com/26656571/114501917-832aab00-9bf0-11eb-8b33-18c41ec7b52e.png)

**Comment by AndreasK79** [comment, marked as answer]
Are you running the latest codebase? From your screenshot, it doesn't look like you are. I would start there.

**Reply by Halfsat** [reply]
Probably not. But I am having a hard time figuring out how. It's in a Docker Container on my NAS, but that hasn't been updated in over a year. I'm thinking an apt-get within the container terminal would do it?

**Reply by magicbug** [reply]
You'll need to refer to the docker container information, Cloudlog isn't provided as an APT package but you'll need to update the codebase to be able to use the new features.
